### DISCUSSION TITLE
Turbo Preset by default

### DISCUSSION THREAD

**Discussion by wisdombox44** [lllyasviel/Fooocus#2181] [General]
"Thank you for the great work. Could you include a default preset for Turbo models? I know it sounds very easy to 'Create Your Own' for developers, but for an average user, it is hard. I have been looking through forums and reading previous discussions, but I did not understand. Finally, someone made a tutorial on YouTube. It's funny; I spent more than one hour trying to succeed because I have zero programming knowledge. I did not realize I made a mistake on one line, which is why it did not work. Finally, it works. If anyone like me wants to try, the YouTube channel name is 'Jump Into AI'."

**Comment by xhoxye** [comment]
Is this what you're looking for?
https://github.com/lllyasviel/Fooocus/pull/1580

**Reply by wisdombox44** [reply]
Thank you, But I don't know what to do with that code. I have zero coding knowledge. I follow that YouTube tutorial works fine for me. Thank you for your reply  :)

**Reply by xhoxye** [reply]
You don't have to deal with code, just download json files and model files
